packages/components-js/tests/vrt/playwright/specs/custom/form-fields.vrt.ts

```diff
@@ -47,19 +48,19 @@ const scenario = async (page: Page, theme: Theme): Promise<void> => {
   <${tag} label="Readonly">
     ${childReadonly}
   </${tag}>
-  <${tag} label="Disabled">
+  <${tag} label="Disabled" ${tag === 'p-pin-code' && 'disabled="true"'}>
```

Comment:
Maybe extract this instead of repeating 3 times, also this will produce an attribute called `false` since we are not in jsx here but string interpolation. Trailing space could also be removed if attribute is not applied.